github4s/shared/src/main/scala/github4s/api/PullRequests.scala

```diff
@@ -120,40 +127,46 @@ class PullRequests[C, M[_]](
   }
 
   /**
-   * List pull request reviews.
-   *
-   * @param accessToken Token to identify the authenticated user
-   * @param headers Optional user header to include in the request
-   * @param owner Owner of the repo
-   * @param repo Name of the repo
-   * @param pullRequest ID number of the PR to get reviews for.
-   */
+    * List pull request reviews.
+    *
+    * @param accessToken Token to identify the authenticated user
+    * @param headers Optional user header to include in the request
+    * @param owner Owner of the repo
+    * @param repo Name of the repo
+    * @param pullRequest ID number of the PR to get reviews for.
```

Comment:
Could you please add the parameter description here?

Reply:
Done.